authorizer-app-backend/src/main/java/org/radarbase/authorizer/service/OAuth2RestSourceAuthorizationService.kt

```diff
@@ -102,7 +102,11 @@ class OAuth2RestSourceAuthorizationService(
         throw HttpBadRequestException("", "Not available for auth type")
     }
 
-    private fun post(form: FormBody, sourceType: String): Request {
+    override fun signUrl(user: RestSourceUser, url: String, method: String, params: MutableMap<String, String?>): String {
+        return ""
```

Comment:
perhaps this could throw a `HttpBadRequestException`, since this authorization service does not support signing a URL? Or a `HttpNotFoundException` since the `sign` url does not exist for a OAuth 2.0 user?